trade-finance-manager-ui/server/controllers/auth/authSso/unauthenticated-auth.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Response } from 'express';
+import { CustomExpressRequest, InvalidPayloadError } from '@ukef/dtfs2-common';
+import { isVerifiedPayload } from '@ukef/dtfs2-common/payload-verification';
+import { EntraIdAuthCodeRedirectResponseBody } from '../../../types/entra-id';
+import { EntraIdAuthCodeRedirectResponseBodySchema } from '../../../schemas';
+
```

Comment:
Can we rename the `authSso` directory to `auth-sso` or maybe just `sso`?

Reply:
Renamed